pkg/kubelet/util/manager/watch_based_manager_test.go

```diff
@@ -144,6 +149,9 @@ func TestSecretCache(t *testing.T) {
 	if err == nil || !strings.Contains(err.Error(), "not registered") {
 		t.Errorf("unexpected error: %v", err)
 	}
+
+	// empty namespace should result in a key equal to the object name
+	assert.Equal(t, "name", store.key("", "name"))
```

Comment:
last thing - can you please explain why you want to make this check here? Should there be a separate test for `store.key` function instead? It seems out of place

Reply:
I just wanted to cover the case when querying a key from the store with en empty string as namespace. I thought it would be a 3 lines test so I felt it was cleaner to add it here

Reply:
it confuses me, as I am trying to understand how this is related to the secret cache. It is more code, but easier to understand later if it is separate

Reply:
I've removed that part